# Data we Accept

Cellxgene is a repository of single cell data built to maximize data reuse by making data visually explorable and standardizing the data's format and schema to facilitate combination and integration of cells from different sources. This page describes the data we can currently accept, and the data we are working to accept in the future:

| Characteristic | Accepted Now | Future | Never | 
| :--- | :--- | :--- | :--- |
| Data type | Derived results in the form of (observation, feature) e.g. (cell, gene) | | Raw data (fastq, bam, hybridization images) |
| Organism | Human, Mouse, SARS-CoV2 | Any organisms with ENSEMBL annotations | Organisms lacking standard feature identifiers |
| Disease | > 20% of cells MUST be Healthy/Normal. Remaining may be from diseased samples | | |
| Sample Type | > 50% of cells must be from tissue samples or organoids | | Cell line datasets |
| Open Sharing | Submitted data must be publicly accessible and contain no PII | | Controlled access data |

If you have data that match these characteristics, the data portal can help amplify its reuse! please see [Upload and Publish Data](./publishing.md) for instructions on how to submit data to cellxgene.
